src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -96,14 +96,23 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+	boolean isExists = true;
```

Comment:
seems like you can rewrite these four lines as `boolean isExists = path.toFile().exists()`?